blosc/fastcopy.c

```diff
@@ -127,15 +108,6 @@ static inline unsigned char *copy_32_bytes(unsigned char *out, const unsigned ch
   chunk = _mm_loadu_si128((__m128i*)from);
   _mm_storeu_si128((__m128i*)out, chunk);
   out += 16;
-#elif !defined(BLOSC_STRICT_ALIGN)
-  *(uint64_t*)out = *(uint64_t*)from;
-  from += 8; out += 8;
-  *(uint64_t*)out = *(uint64_t*)from;
-  from += 8; out += 8;
-  *(uint64_t*)out = *(uint64_t*)from;
-  from += 8; out += 8;
-  *(uint64_t*)out = *(uint64_t*)from;
-  from += 8; out += 8;
 #else
   int i;
   for (i = 0; i < 32; i++) {
```

Comment:
use use `memcpy()`

Reply:
IMHO this clean up does not need to be done in this patch. blosc2 maintainers are probably in a better position to follow up and fix these:)

But I don't mind if thesamesam has cleaned up this part as well:)